sdk/appconfiguration/azure-appconfiguration/samples/conditional_operation_async_sample.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: conditional_operation_async_sample.py
+DESCRIPTION:
+    This sample demos conditional set/get/delete operations for app configuration
+USAGE: python conditional_operation_async_sample.py
+"""
+
+import os
+import sys
+from azure.core import MatchConditions
+from azure.core.exceptions import ResourceModifiedError
+import asyncio
+from azure.appconfiguration import ConfigurationSetting
+from azure.appconfiguration.aio import AzureAppConfigurationClient
+from util import print_configuration_setting
+
+async def main():
+    try:
+        CONNECTION_STRING = os.environ['AZURE_APPCONFIG_CONNECTION_STRING']
+
+    except KeyError:
+        print("AZURE_APPCONFIG_CONNECTION_STRING must be set.")
+        sys.exit(1)
```

Comment:
As long as there is going to be a `util` module for `print_configuration_setting` why not put this boilerplate for getting the conn str in a function  in there as well?